ui/pages/confirmations/components/confirm/row/dataTree.tsx

```diff
@@ -11,6 +11,7 @@ import { getTokenStandardAndDetails } from '../../../../../store/actions';
 
 import { Box } from '../../../../../components/component-library';
 import { BlockSize } from '../../../../../helpers/constants/design-system';
+import * as hookModule from '../../../../../hooks/useAsyncResult';
```

Comment:
Minor, but why the `as` import?

Reply:
ah I was copying over the way it was used in a test file and forgot to update. I would also prefer importing the methods directly. Thanks for the callout! Will see if I can merge this PR and then update this in a follow-up PR

Reply:
Nice catch! Missed this when I was copying over the import from another test file. Updated [d4e60ac](https://github.com/MetaMask/metamask-extension/pull/26324/commits/d4e60ac556a3d2ec5ed1977cd9614f3d8ece7d46)